src/opf/socwr.jl

```diff
@@ -223,30 +223,20 @@ function extract_dual(opf::OPFModel{OPF}) where {OPF <: Union{SOCOPFQuad,SOCOPF}
         "ohm_pt"     => zeros(T, E),
         "ohm_qf"     => zeros(T, E),
         "ohm_qt"     => zeros(T, E),
-        "va_diff_lb" => zeros(T, E),
-        "va_diff_ub" => zeros(T, E),
+        "va_diff"    => zeros(T, E),
```

Comment:
Note that I also merged duals of voltage angle difference constraint, using the same convention.
This matches the data format used for DC/AC